runtime/lua/vim/filetype.lua

```diff
@@ -929,10 +929,10 @@ local filename = {
   ["exim.conf"] = "exim",
   exports = "exports",
   [".fetchmailrc"] = "fetchmail",
-  fvSchemes = function() vim.fn["dist#ft#FTfoam"]() end,
-  fvSolution = function() vim.fn["dist#ft#FTfoam"]() end,
-  fvConstraints = function() vim.fn["dist#ft#FTfoam"]() end,
-  fvModels = function() vim.fn["dist#ft#FTfoam"]() end,
+  fvSchemes = function(path, bufnr) return require("vim.filetype.detect").foam(bufnr) end,
```

Comment:
Should I rename the unused `path` argument to `_`?

Reply:
No; please keep to the patterns in `filetype.lua` -- things should be consistent; they can still be improved (en masse) afterwards.